example/speech-demo/decode_mxnet.sh

```diff
@@ -70,7 +70,7 @@ feats="scp:$sdata/JOB/feats.scp"
 inputfeats="$sdata/JOB/mxnetInput.scp"
 
 
-if [ ! -f $sdata/1/feats.scp ]; then
+if [ -f $sdata/1/feats.scp ]; then
```

Comment:
I think it was meant to avoid re-constructing mxnetinput.scp, so should be ! -f mxnetinput.scp?